src/DatadogTestLogger/DatadogTestResultSerializer.cs

```diff
@@ -59,19 +60,52 @@ public string Serialize(LoggerConfiguration loggerConfiguration, TestRunConfigur
 
     private void ShowEnvironmentVariables(StringBuilder builder, string title)
     {
+        const string defaultRegexPattern = @"^(DD_|COR_|CORECLR_).*";
```

Comment:
Should these be case insensitive? 🤔

Reply:
good one, done in [8c0ba34](https://github.com/tonyredondo/datadog-test-logger/pull/19/commits/8c0ba342717167f43daaf15f3fbe97fe61560233)